docker/smqtk_services_2/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM kitware/smqtk
+MAINTAINER omar.padron@kitware.com
+
+# Add apt repo for postgres
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys \
```

Comment:
Yes, since the Ubuntu repo that normally provides the client is removed from the docker image.